src/unity/python/turicreate/toolkits/one_shot_object_detector/util/_augmentation.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+def preview_augmented_images(data, target, n=1):
```

Comment:
Probably this should share a lot of the same interface with the `create` method.